.github/workflows/docker-publish.yml

```diff
@@ -33,9 +33,9 @@ jobs:
 
       - name: Install cosign
         if: github.event_name != 'pull_request'
-        uses: sigstore/cosign-installer@f3c664df7af409cb4873aa5068053ba9d61a57b6 #v2.6.0
+        uses: sigstore/cosign-installer@9becc617647dfa20ae7b1151972e9b3a2c338a2b #v2.8.1
```

Comment:
Just for added context, why are we using a commit hash as opposed to a tag?

Reply:
That's just the way they had it defined in the template so I kept it.  I can change it if you think that would be better